src/chainstate/stacks/index/bits.rs

```diff
@@ -150,35 +143,12 @@ pub fn get_ptrs_byte_len(ptrs: &[TriePtr]) -> usize {
     node_id_len + TRIEPTR_SIZE * ptrs.len()
 }
 
-/// Encode a Trie node's child pointers as a byte string by appending them to the given buffer.
+/// Helper to determine how many bytes a Trie node's child pointers will take to encode for consensus.
 #[inline]
-pub fn ptrs_to_bytes(node_id: u8, ptrs: &[TriePtr], buf: &mut Vec<u8>) -> () {
-    assert!(check_node_id(node_id));
-    assert_eq!(node_id_to_ptr_count(node_id), ptrs.len());
-
-    buf.push(node_id);
-
-    // In benchmarks, this while() loop is noticeably faster than the more idiomatic "for ptr in ptrs.iter()"
-    let mut i = 0;
-    while i < ptrs.len() {
-        ptrs[i].to_bytes(buf);
-        i += 1;
-    }
-}
-
-/// Encode only the consensus-relevant bits of a Trie node's child pointers to the given buffer.
-#[inline]
-pub fn ptrs_to_consensus_bytes(node_id: u8, ptrs: &[TriePtr], buf: &mut Vec<u8>) -> () {
-    assert!(check_node_id(node_id));
-
-    buf.push(node_id);
-    
-    // In benchmarks, this while() loop is noticeably faster than the more idiomatic "for ptr in ptrs.iter()"
-    let mut i = 0;
-    while i < ptrs.len() {
-        ptrs[i].to_consensus_bytes(buf);
-        i += 1;
-    }
+pub fn get_ptrs_consensus_byte_len(ptrs: &[TriePtr]) -> usize {
+    let node_id_len = 1;
+    let consensus_trie_ptr_size = 2 + 32;// 2: id + chr, 32: block header hash
```

Comment:
We have a constant `TRIEHASH_ENCODED_SIZE` that you can use here instead of `32`.

Reply:
Addressed in eb57cc2